{"name": "test_new_customer_a1_error_3", "status": "passed", "description": "新增账户开户失败的用例", "attachments": [{"name": "log", "source": "bda7a3d6-08da-41c5-9b5b-05aecb79d4f2-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "03c28f0d-c6b9-4294-aa50-c2c968e2dac7-attachment.txt", "type": "text/plain"}], "start": 1617349324299, "stop": 1617349325701, "uuid": "f6c5edec-9ea9-4770-9e1e-f248c8858a41", "historyId": "3c2b3aaf45305d32372fea8c95a95602", "testCaseId": "3a4350af7273cadb0b49666b67675833", "fullName": "tests.test_a6_newclientregister.TestNewCustomerAccount#test_new_customer_a1_error_3", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_a6_newclientregister"}, {"name": "subSuite", "value": "TestNewCustomerAccount"}, {"name": "host", "value": "DESKTOP-U13POE8"}, {"name": "thread", "value": "6480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_a6_newclientregister"}]}